frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -7,6 +7,7 @@
 
 action.GitMachete.description.disabled.undefined.branch-name={0} disabled due to undefined branch name
```

Comment:
Also while you're here: `Open Git Machete` description/name should rather be `Open Git Machete (VCS?) Tab` for clarity

![image](https://user-images.githubusercontent.com/3383210/88684355-0c4a9200-d0f5-11ea-80fb-ee2b815b9850.png)